### DISCUSSION TITLE
Forward Node Grid Status is Fault

### DISCUSSION THREAD

**Discussion by aa18afr** [Security-Onion-Solutions/securityonion#11520] [Unsupported Versions] [answered]
Hi,

The grid shows the fault status for one of the sensor/forward nodes. the logs are checked, netcat connectivity is also checked, and using so-test packets were received from the sensor/forward node at the manager node. I have checked previously reported issues related to the grid but still the issue persists. Please provide any resolution, thank you.

**Manager node ip:** 192.168.150.187
**Sensor/forward node ip:** 192.168.150.166

The following image shows the connectivity status:
![ping and netcat - sensor to mgr](https://github.com/Security-Onion-Solutions/securityonion/assets/142659986/065ac034-a1da-4e86-b2eb-4a9c40dd9d5e)

The following image shows the live log status received on the manager node end:
![live response from sensor_b](https://github.com/Security-Onion-Solutions/securityonion/assets/142659986/204a1ebd-e697-4693-b01a-487945346d36)

The following image shows the grid status: 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/142659986/ac782b77-cd14-4118-8a41-fe1c4dc6d8c0)

The following image shows the result of the so-test where the manager node is receiving so-test results:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/142659986/f659c5d1-0b78-466e-a8aa-80f1e11d11e2)

**Comment by robbiemarshall** [comment]
Have you tried running a so-checkin to see if that clears the fault up?

**Reply by aa18afr** [reply]
Yes, I tried but I couldn't find anything which can remove the FAULT status in grid.

**Reply by aa18afr** [reply, marked as answer]
The issue was at the hardware level where clock was not synced and is resolved via syncing clocks of hardware OS.
